cmake/SeracConfigHeader.cmake

```diff
@@ -36,7 +36,7 @@ message(STATUS "Configuring Serac version ${SERAC_VERSION_FULL}")
 #------------------------------------------------------------------------------
 # Create variable for every TPL
 #------------------------------------------------------------------------------
-set(TPL_DEPS ASCENT AXOM CONDUIT FMT HDF5 MFEM MPI TRIBOL CALIPER PETSC )
+set(TPL_DEPS ASCENT AXOM CONDUIT FMT HDF5 MFEM MPI TRIBOL CALIPER PETSC RAJA UMPIRE CHAI)
```

Comment:
I see chai being used in this PR, but not raja or umpire. Are they dependencies of chai, or just being added now, expecting to be used later on?

Reply:
They are dependencies of chai that were added a while back in preparation for future GPU work - the changes here were not needed until now because we never actually used raja/umpire (or something that depended on it)